conf/application.conf

```diff
@@ -143,20 +143,82 @@ personal-tax-account {
   path = "/personal-account"
 }
 
-two-step-verification {
-  enabled = false
-  host = ${company-auth.host}
-  path = "/coafe/two-step-verification/register"
-}
-
 two-step-verification-required {
   host = ${business-tax-account.host}
   path = "/business-account/two-step-verification/failed"
 }
 
+user-delegation-frontend {
+  host = "http://localhost:9851"
+  path = "/user-delegation"
+}
+
+self-assessment-enrolments = ["enr3", "enr4"]
+vat-enrolments = ["enr5", "enr6"]
+
 company-auth.host = "http://localhost:9025"
 contact-frontend.host = "http://localhost:9250"
 tax-account-router.host = "http://localhost:9280"
-
 business-enrolments = "enr1,enr2"
-self-assessment-enrolments = "enr3,enr4"
+
+locations {
+  two-step-verification-optional {
+    name = "two-step-verification"
+    url = ${two-step-verification.host}${two-step-verification.path}
+    queryparams {
+      continue = ${locations.bta.url}
+      failure = ${locations.bta.url}
+      origin = "business-tax-account"
+    }
+  }
+  two-step-verification-mandatory {
+    name = "two-step-verification"
+    url = ${two-step-verification.host}${two-step-verification.path}
+    queryparams {
+      continue = ${locations.bta.url}
+      failure = ${locations.tax-account-router.url}
+      origin = "business-tax-account"
+    }
+  }
+  set-up-extra-security {
+    name = "set-up-extra-security"
+    url = ${user-delegation-frontend.host}${user-delegation-frontend.path}"/set-up-extra-security"
+  }
+  bta {
+    name = "business-tax-account"
+    url = ${business-tax-account.host}${business-tax-account.path}
+  }
+  tax-account-router{
+    name = "tax-account-router"
+    url = ${tax-account-router.host}"/account"
+  }
+}
+
+two-step-verification {
+  enabled = false
+  host = ${company-auth.host}
+  path = "/coafe/two-step-verification/register"
+  user-segment {
+    sa {
```

Comment:
We need a name for the rule for auditing and analytics purposes The key serves the name of the rule itself. I feel the order doesn't matter here because the segments will be mutually exclusive. No user will belong to two different segments at the same time.